changelog.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Added
+- [[#2079] Add example number 1](https://github.com/ballerina-platform/ballerina-standard-library/issues/2079)
```

Comment:
I think we don't need to updated the change log as this is not a functional change.

Reply:
Removed.